### DISCUSSION TITLE
Conditional render in items table

### DISCUSSION THREAD

**Discussion by Jerome-thinking** [frappe/print_designer#197] [General]
Within our item table we have some information that should only show on certain conditions. E.g. we provide rentable items, that need to show information about the rental duration. "Normal items" (like our service of installing the stuff we rent) does not need this information. How can we achieve that this particular information (custom_duration and custom_duration_uom resp. row.custom_duration and custom_duration_uom) only show when the item's rentable-flag is set and the duration is greater than 0?

**Comment by maharshivpatel** [comment]
@Jerome-thinking you can use jinja to conditionally render elements. https://github.com/frappe/print_designer/pull/99

**Reply by mrmm** [reply]
How can you do that please ?
